conda_forge_tick/migrators.xsh

```diff
@@ -392,7 +392,6 @@ class Noarch(Migrator):
     sel_pat = re.compile('(.+?)\s*(#.*)?\[([^\[\]]+)\](?(2)[^\(\)]*)$')
     unallowed_reqs = ['toolchain', 'gcc', 'cython', 'clangdev']
     checklist = ['No compiled extensions',
-                 'No post-link or pre-link or pre-unlink scripts',
```

Comment:
I don't know if we should remove this, even if the bot does check. There might be some reason the bot missed the scripts, and we want maintainers to be sure the PR can be merged.